docs/getting-started/getting-started.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Getting started
+---
+
+# Getting started
+
+Patrol consists of 2 packages:
+
+- [patrol] – the core package
+- [patrol_cli] - the CLI app which enables the [native automation feature]
+
+There are two ways of using Patrol - to create **E2E UI tests** which contain 
+testing **native features of your app** (eg. localization permissions) or to **enhance** your experience of writing 
+**widget tests** by making use of our **custom finders**. Depending of which way you want to go, 
+head over to [native setup] or [custom finders setup].
```

Comment:
This paragraph sounds like these choices are mutually exclusive (when they're complementary).